src/main/java/http/Header.java

```diff
@@ -0,0 +1,54 @@
+package http;
+
+import com.github.jknack.handlebars.internal.lang3.StringUtils;
+import http.Const.HttpConst;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created By kjs4395 on 2020-06-05
+ */
+public class Header {
+    private Map<String, String> headers;
+
+    public Header() {
+        this.headers = new HashMap<>();
+    }
+
+    public void addHeaderValue(String headerLine) {
+        String[] values = headerLine.split(HttpConst.HEADER_SEPARATOR,2);
+
+        if(values.length < 2) {
+            throw new IllegalArgumentException("Invalid header format");
+        }
+
+        this.headers.put(values[0].trim(), values[1].trim());
+    }
+
+    public void addKeyAndValue(String key, String value) {
+        if(this.headers == null) this.headers = new HashMap<>();
```

Comment:
if 문에서는 항상 중괄호를 쓰는것을 권장하고 있습니다.
https://www.oracle.com/java/technologies/javase/codeconventions-statements.html#449
```suggestion
        if (this.headers == null) {
                this.headers = new HashMap<>();
        }
```